src/ResourceManager/LogicApp/Commands.LogicApp/Utilities/IntegrationAccountControlNumber.cs

```diff
@@ -0,0 +1,131 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.LogicApp.Utilities
+{
+    using System;
+    using System.Management.Automation;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// LogicApp client partial class for integration account control number types.
+    /// </summary>
+    public partial class IntegrationAccountClient
```

Comment:
Because the types are client-side only. The swagger and SDK specify what the server is aware of, which here is just an obscure/opaque JToken object. The clients store state specific to themselves within the session. Different client-defined types can and are stored in different sessions.